app/controllers/api/v1/forecast_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::ForecastController < ApplicationController
+  def index
+    location = Location.find_long_lat(forecast_params[:location])
```

Comment:
Note to self: `find_long_lat `grabs the city, state, country, latitude and longitude for a location, then passes that information to the forecast search.

This logic took me a while to wrap my head around.